yt/frontends/athena_pp/data_structures.py

```diff
@@ -278,16 +278,9 @@ def _set_code_unit_attributes(self):
             mylog.warning("Assuming 1.0 = 1.0 %s", cgs)
             setattr(self, "%s_unit" % unit, self.quan(1.0, cgs))
 
-    def set_code_units(self):
-        super(AthenaPPDataset, self).set_code_units()
```

Comment:
Why aren't we calling `set_code_units` on the superclass? For example, will `_set_code_unit_attributes` still be called without this?